internal/infrastructure/persistence/sqlite/queries/history.sql

```
@@ -1,18 +1,46 @@
+-- name: GetRecentHistoryWindow :many
+SELECT * FROM history
+WHERE last_visited < @before AND last_visited >= @after
+ORDER BY last_visited DESC;
+
+-- name: GetRecentHistoryWindowByDomain :many
+SELECT * FROM history
+WHERE domain = @domain AND last_visited < @before AND last_visited >= @after
+ORDER BY last_visited DESC;
+
+-- name: HasHistoryBefore :one
+SELECT EXISTS(SELECT 1 FROM history WHERE last_visited < @before LIMIT 1);
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)